Exercise_C.js

```diff
@@ -15,4 +15,21 @@ See README.md if you need help running your code.
 
 for (let i = 1; i <= 30; i++) {
   // CODE HERE
+  let num = [];
+  if(i % 3 == 0 && i % 5 == 0){
+    num.push("Hong Kong")
+    console.log(num)
+  } else if(i % 3 == 0){
+    num.push("Hong")
+    console.log(num)
+  } else if(i % 5 == 0){
+    num.push("Kong")
+    console.log(num)
+  } else {
+    num.push(i)
+    console.log(num)
+  }
+
 }
+
+

```

Comment:
For this question, you do not have to place it inside an array. You can console.log the result outside